sympy/stats/crv_types.py

```diff
@@ -48,21 +48,21 @@
 
 from __future__ import print_function, division
 
-from sympy import (log, sqrt, pi, S, Dummy, Interval, sympify, gamma,
-                   Piecewise, And, Eq, binomial, factorial, Sum, floor, Abs,
-                   Lambda, Basic, lowergamma, erf, erfc, erfi,  erfinv, I,
-                   hyper, uppergamma, sinh, atan, Ne, expint, Integral)
+import random
```

Comment:
was it necessary to change all these lines? it's not very comfortable for reviewing.